src/diffusers/models/autoencoders/autoencoder_kl_cogvideox.py

```diff
@@ -1057,6 +1057,7 @@ def __init__(
         force_upcast: float = True,
         use_quant_conv: bool = False,
         use_post_quant_conv: bool = False,
+        invert_scale_latents: bool = False,
```

Comment:
I don't think this change is needed - we can just adjust when we scale it, ie.g instead of `* scale_factor`, we do * `1/scale_factor`

Reply:
For the original CogVideoX 1.0 models, we need to do `latents * scale_factor`, but for the new version we need to do `latents / scale_factor` in `prepare_latents` IIUC from my conversation with Yuxuan. So, I think we might need it :( I will check again

Reply:
This part is necessary because there is indeed a difference in execution between 1.5 and 1.0 here. The training process for 1.5 forgot to use *scale_factor.

Reply:
why cannot we just update the scale_factor here? for example if it is 5 , we just use 1/5 instead

Reply:
we cannot update the `decode_latents` too? it is creating inconsistency, no?

Reply:
So, one way or the other, we will need to determine when to multiply by scale factor and when to divide. The encode method needs to support both (multiply for 1.0, divide for Cog 1.5). In the decode method, we just need to support divide (already exists).

Even if we invert the scale factor during conversion of VAE, we still need to be able to determine which version of the model is running in order for the encode method to work as expected. As of now, we can do this by a check like `if self.transformer.config.patch_embed_t is None: then version 1.0 else version is 1.5`, but I don't think this is a nice way to handle it.